config/deprecation-workflow.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+
+window.deprecationWorkflow = window.deprecationWorkflow || {};
+window.deprecationWorkflow.config = {
+  workflow: [
+    { handler: 'silence', matchId: 'ember-global' },
+    { handler: 'silence', matchId: 'manager-capabilities.modifiers-3-13' },
+    { handler: 'silence', matchId: 'ember.built-in-components.import' },
+    { handler: 'silence', matchId: 'this-property-fallback' },
+  ],
+};
```

Comment:
You have to manually update this config which kind of sucks. 
1. Run the tests suite or navigate through the app
2. Run `deprecationWorkflow.flushDeprecations()` in the console
3. Copy the output, and paste it in here.

If you fix one of the deprecations, re-run the above, and update this file